kratos/utilities/dynamic_proxies.h

```diff
@@ -0,0 +1,110 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Máté Kelemen
+//
+
+#pragma once
+
+// Project includes
+#include "includes/global_variables.h" // Globals::DataLocation
+#include "includes/model_part.h" // ModelPart::NodesContainerType, ModelPart::ElementsContainerType, ModelPart::ConditionsContainerType
+#include "utilities/proxies.h" // EntityProxy, ContainerProxy
+#include "includes/kratos_export_api.h" // KRATOS_API
+
+// System includes
+#include <variant> // variant
+
+
+namespace Kratos {
+
+
+class KRATOS_API(KRATOS_CORE) DynamicEntityProxy
```

Comment:
I would suggest to move this entire thing to pybind or delete it fully. Because, using this in cpps will have a big perfromance hit.

Reply:
deleted it